.github/workflows/yaml-lint-test.yml

```diff
@@ -0,0 +1,26 @@
+name: YAML Lint
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      
+    - name: Install yamllint 
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y yamllint
+
+    - name: Lint YAML files
+      run: |
+       find . -name '*.y*ml' -exec yamllint -p {} \+
```

Comment:
Shouldn't we limit this to files only and exempt the `ansible/` sub-directory from YAML linting because this is already done by `ansible-lint` implicitly (`ansible-lint` utilizes `yamllint`) and potentially would just result in duplicate errors and warnings.

The name spec `'*.y*ml'` isn't very precise and might potentially also would match non-YAML files. Most likely we only want to lint `*.yaml` files (which is the official file extension) and `*.yml` (which is an accepted file extension).

Last but not least, `yamllint` as of version 1.35.1 doesn't have a `-p` flag.

To sum it up, I guess
```
find . -type f ! -wholename './ansible/*' \( -name \*.yaml -o -name \*.yml \) -exec yamllint {} \+
```
comes closer to what we need.

Another option might be introducing a [`.yamllint`](https://yamllint.readthedocs.io/en/stable/configuration.html) file and add the `ansible/` sub-directory to the [ignore](https://yamllint.readthedocs.io/en/stable/configuration.html#ignoring-paths) list. In this case we can get rid of `find` entirely and just let `yamllint` recurse on its own over the entire `github.workspace`.